manifests/e/EclipseFoundation/SUMO/1.11.0.1405/EclipseFoundation.SUMO.locale.en-US.yaml

```
@@ -0,0 +1,29 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=NVS1.7-2-5
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: EclipseFoundation.SUMO
+PackageVersion: 1.11.0.1405
+PackageLocale: en-US
+Publisher: Eclipse Foundation AISBL
```

```suggestion
Publisher: Eclipse
```
This must match Control Panel.